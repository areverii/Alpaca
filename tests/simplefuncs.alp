function void print_3(int input):
    printi(5000000000)
end

function void main(<string> args):
    print_3(2)
    prints("String!")
    printf(3.453)
end
