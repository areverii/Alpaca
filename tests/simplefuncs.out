5000000000
String!
3.453
